assets/js/protocol/protocol-navigation.js

```diff
@@ -175,13 +177,51 @@ if (typeof window.Mzp === 'undefined') { // eslint-disable-line block-scoped-var
             }
         }
     };
+    /**
+     * use Intersection Observer API to keep track of when the mobile
+     * nav menu is displayed to handle aria roles better
+     */
+    Navigation.menuButtonVisible = function (callback) {
+        // check if Intersection observer is supported
+        if (!('IntersectionObserver' in window) ||
+            !('IntersectionObserverEntry' in window) ||
+            !('intersectionRatio' in window.IntersectionObserverEntry.prototype)) {
+            // load polyfill now
```

Comment:
Question: Is there anything that should happen in this condition here? Adding a global polyfill for this feels like a bit too much perhaps. Are we just degrading gracefully instead, or should something else need to happen still? Maybe updating the comment here to that effect might be enough :)

Reply:
I think just degrading gracefully - will update the coment